unime/src/routes/(app)/me/+page.svelte

```diff
@@ -166,10 +177,29 @@
       <!-- Skipped onboarding journey -->
       <div class="flex grow flex-col items-center justify-center">
         <IconMessage icon={Ghost} title={$LL.ME.EMPTY_CREDENTIALS.TITLE()} />
-        <p class="w-[280px] pt-[15px] text-center text-[13px]/[24px] font-normal text-slate-500 dark:text-slate-300">
-          {$LL.ME.DEMO.TEXT_1()} <span class="font-semibold text-primary">https://demo.ngdil.com</span>
-          {$LL.ME.DEMO.TEXT_2()}
-        </p>
+        <div class="w-[280px] pt-[15px] text-center text-[13px]/[24px] font-normal text-slate-500 dark:text-slate-300">
+          {$LL.ME.DEMO()}
+          <div class="flex flex-col gap-3 pt-[15px]">
```

Comment:
There should be only one SVG for each icon. The SVG can adapt it's styling based on the global `dark` class.

Reply:
Refactored 👍 Thanks for pointing it out!